lib/python/rosie/svn_pre_commit.py

```diff
@@ -260,7 +264,8 @@ def run(self, repos, txn):
 
             # Only admin users can remove the suite
             if author not in admin_users and not path_tail:
-                bad_changes.append(BadChange(status, path))
+                msg = "Only admin users can remove the suite"
```

Comment:
```admin_users``` includes the suite owner who is the only person allowed to remove a suite under normal circumstances. Therefore, I think the message should be:
```"Only the suite owner can remove the suite"```

Reply:
Done